### DISCUSSION TITLE
🔗 Redirects from default locale to root path. (avoid page duplicate)

### DISCUSSION THREAD

**Discussion by ProjectINT** [vercel/next.js#20219] [Help]
I have sub path localization. Default locale 'en'
1. First strange thing is that i have both routes: / and /en. (this is page duplicate)
2. I want to redirect all /en/* to /*. How i can do this?

I'm try 2 varians: (next.config.js redirects)

` {
     source: '/en/boards',
     destination: '/boards',
     permanent: true,
  },
`
this variant does not effect. And its expected, because next understand  "/en/boards" as "/en/en/boards", and really redirect /en/en/boards to /en/boards (but why NOT /boards ?)
`{
    source: '/en/boards',
    destination: '/boards',
    locale: false,
    permanent: true,
},
`
this variant does not work. pages not loads, any errors not returns.

**Comment by dimisus** [comment]
Yep, huge issue, still persists by the time writing.

**Comment by ProjectINT** [comment]
I'm solved this problem by adding rel="canonical", but really strange why this problem not attract attention

**Comment by dimisus** [comment]
Here is another issue but within the sandbox example it worked as I would expect it to:

https://github.com/vercel/next.js/discussions/19482

**Reply by dimisus** [reply]
https://github.com/vercel/next.js/issues/21040#issuecomment-816785857

Moved back to my custom server and proper regex redirects
